filebeat/docs/modules/elasticsearch.asciidoc

```diff
@@ -70,8 +75,14 @@ Example config:
 ----
   audit:
     var.paths:
-      - /var/log/elasticsearch/*_audit.json
+      - /var/log/elasticsearch/*_access.log  # Plain text logs
+      - /var/log/elasticsearch/*_audit.log   # JSON logs
```

Comment:
Yes, this was the case in 6.7, I believe. Starting 7.0 the JSON audit logs were put into `*_audit.json`. I think, to avoid confusion and complexity, I'll just remove this line altogether.

Reply:
Removed in f062d3ee4c8319d04d5e88bd277eed1617471986.